code/modules/mob/living/silicon/robot/robot_mob.dm

```diff
@@ -1420,6 +1430,14 @@ GLOBAL_LIST_INIT(robot_verbs_default, list(
 	else
 		REMOVE_TRAITS_IN(src, LOCKDOWN_TRAIT)
 
+// Allows the borg to unlock themselves after a lengthy period of time.
+/mob/living/silicon/robot/proc/lockdown_override()
+	SetLockdown(FALSE)
+	message_admins("<span class='notice'>The lockdown on cyborg [src] has been lifted because their lockdown timer expired.</span>")
+	to_chat(src, "<br><br><span class='notice'>Lockdown subsystem override successful.")
```

Comment:
```suggestion
	to_chat(src, "<br><br><span class='notice'>Lockdown subsystem override successful.</span>")
```
Are the `<br><br>` doing anything here?

Reply:
They push the message down away from the last sent message in chat to make it more prominent. The notices being sent to the AI have this characteristic too.